README.md

```diff
@@ -587,6 +588,14 @@ Looking for quadlet examples files? See our [docs dir](docs/quadlet-examples/).
 
 If your looking for contribute to the project use our [development README guide](docs/devel/README.md) as start point.
 
+## Realtime
+
+To enable real-time removal of sched_* blockage via seccomp, comment out the SeccompProfile.
+
+```bash
```

Comment:
IT should be qm-container,
And it should be in quadlet dropin with ovveride 
this one stays as it is but the dropin should have the following 

[Container]
SeccompProfile=""

maybe something like that

```
cat << EOF >> /etc/containers/systemd/qm.container.d/rt.conf
> [Container]
SeccompProfile=""
> EOF
```